bin/run-in-docker.sh

```diff
@@ -33,14 +33,13 @@ output_dir=$(realpath "${3%/}")
 mkdir -p "${output_dir}"
 
 # Build the Docker image
-docker build --rm -t exercism/test-runner .
+docker build --rm  -t exercism/ballerina-test-runner .
 
 # Run the Docker image using the settings mimicking the production environment
 docker run \
     --rm \
-    --network none \
-    --read-only \
```

Comment:
Why did you remove these flags? The second one we don't yet enforce, but the first is a hard requirement. Could you re-add at least the first one and make sure the test runner still works?

Reply:
Ballerina pulls in dependencies and was failing when the network mode was set to `none`. Now that everything is working Ill try to work out a builder phase that pulls dependencies.

Reply:
I think I have a working solution for the network issue. But the `--read-only` flag prevents the test script from writing the results.json file. Will it be ok to remove the read-only requirement?

Reply:
Ok, after many tries, I think I got it working with a multi-stage build and lib caching.

Reply:
> But the --read-only flag prevents the test script from writing the results.json file. Will it be ok to remove the read-only requirement?

You should be able to write the results.json to the output directory, which is the third argument that gets passed in.